packages/runtime-core/src/main/kotlin/elide/runtime/core/internals/graalvm/GraalVMEngine.kt

```diff
@@ -75,7 +124,38 @@ import elide.runtime.core.internals.graalvm.GraalVMEngine.Companion.create
       lifecycle: MutableEngineLifecycle,
     ): GraalVMEngine {
       val languages = configuration.languages.map { it.languageId }.toTypedArray()
-      val builder = Engine.newBuilder(*languages).allowExperimentalOptions(true)
+      val builder = Engine.newBuilder(*languages).apply {
+        allowExperimentalOptions(true)
+
+        // base options enabled for every engine
+        enableOptions(
+          "engine.BackgroundCompilation",
+          "engine.UsePreInitializedContext",
+          "engine.Compilation",
+          "engine.MultiTier",
+          "engine.Splitting",
+          "engine.Inlining",
+          "engine.InlineAcrossTruffleBoundary",
+        )
+
+        // isolate options
+        if (ENABLE_ISOLATES) {
+          disableOption("engine.SpawnIsolate")
```

Comment:
should this be `enableOption` instead? I copied this behaviour from [NativeContextManagerImpl](https://github.com/elide-dev/elide/blob/a3b071f0ee62ffdbbd9a8e68f2675c00763331eb/packages/graalvm/src/main/kotlin/elide/runtime/gvm/internals/context/NativeContextManagerImpl.kt#L85)

Reply:
yes, i think it should be `enableOption`; i'm following up in next PR with actual flags for that